.github/workflows/push-pull-main.yml

```diff
@@ -144,11 +117,10 @@ jobs:
         env-prefix: /usr/share/miniconda3/envs/my-env
         env-label: linux-64-py-3-10
         env-files: ${{ inputs.tests-env-files }}
-        test-dir: tests/benchmark
+        test-dir: test_benchmarks
```

Comment:
As discussed in #32, I'm ok with variableizing this, but I absolutely do not want to _encourage_ the cluttered pattern of having multiple tests dirs in the home dir.

And anyhow, you'd then need to go update the README for this, since the expected repo structure is outlined there, and clearly specifies that all the tests should be in subdirs under tests/